ergotree-interpreter/src/eval/sglobal.rs

```diff
@@ -98,4 +196,98 @@ mod tests {
             expected_xor.as_slice()
         );
     }
+
+    #[test]
```

Comment:
Can you add roundtrip tests (for example i64->big-endian byte array->back to i64 using this method) for arbitrary bytes/shorts/etc using proptest? See other proptest examples like in src/eval/downcast.rs

Reply:
Added proptest roundtrip tests following the pattern in downcast.rs.